web/vue-app/src/components/IndexationForm.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div id="indexation-form">
+    <form @submit.prevent="handleSubmit">
+      <div class="groups">
+        <div class="group">
+          <label>Contract Start Date:</label>
+          <input
+            ref="first"
+            type="date"
+            v-model="input.start"
+            :class="{ 'has-error': (submitting && (emptyStart ||conceptError)) }"
+            @focus="clearStatus"
+            @keypress="clearStatus"
+          />
+        </div>
+        <div class="group">
+          <label>Contract Signed On:</label>
+          <input
+            type="date"
+            v-model="input.signed"
+            :class="{ 'has-error': (submitting && (emptySigned || farseerError || conceptError))}"
+            @focus="clearStatus"
+          />
+        </div>
+      </div>
+      <div class="groups">
+        <div class="group">
+          <label>Rent (Euro):</label>
+          <input
+            type="text"
+            v-model="input.rent"
+            :class="{ 'has-error': (submitting && (emptyRent || InvalidRent)) }"
+            @focus="clearStatus"
+          />
+        </div>
+        <div class="group">
+          <label>Region (brussels, flanders, wallonia):</label>
+          <input
+            type="text"
+            v-model="input.region"
+            :class="{ 'has-error': (submitting && (emptyRegion ||invalidLocation)) }"
+            @focus="clearStatus"
+          />
+        </div>
+      </div>
+
+
+      <br>
+      <p v-if="lackOfInputError && submitting" class="error-message">
+        ❗There seems to be some missing fields 😞
+      </p>
+      <p v-if="this.farseerError && submitting" class="error-message">❗One of your your signed date is in the future</p>
+      <p v-if="this.conceptError && submitting" class="error-message">❗Sign date cannot be in the future of the start date 😏 </p>
+      <p v-if="this.invalidRent && submitting" class="error-message">❗Please input a numerical value for the rent 🔎 </p>
+      <p v-if="this.invalidLocation && submitting" class="error-message">❗Region selection can only be one of three: brussels, flanders, wallonia </p>
```

Comment:
```suggestion
      <template v-if="submitting">
        <p v-if="lackOfInputError" class="error-message">
        ❗There seems to be some missing fields 😞
      </p>
        <p v-if="this.farseerError" class="error-message">❗One of your your signed date is in the future</p>
        <p v-if="this.conceptError" class="error-message">❗Sign date cannot be in the future of the start date 😏 </p>
        <p v-if="this.invalidRent" class="error-message">❗Please input a numerical value for the rent 🔎 </p>
        <p v-if="this.invalidLocation" class="error-message">❗Region selection can only be one of three: brussels, flanders, wallonia </p>
      </template>
```
With all the errors in your `v-if` being computed properties

Reply:
👌